cmd/stolonctl/status.go

```diff
@@ -0,0 +1,141 @@
+/*
+Copyright (C) 2018 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+	"time"
+
+	"github.com/gravitational/stolon-app/internal/stolonctl/pkg/cluster"
+
+	"github.com/gravitational/trace"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var statusCmd = &cobra.Command{
+	Use:   "status",
+	Short: "Show status of PostgreSQL cluster",
+	RunE:  status,
+}
+
+func init() {
+	stolonctlCmd.AddCommand(statusCmd)
+}
+
+func status(ccmd *cobra.Command, args []string) error {
+	clusterStatus, err := Status()
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	PrintStatus(clusterStatus)
+	return nil
+}
+
+// Status returns status of Stolon cluster(Pods state and ClusterView)
+func Status() (*cluster.Status, error) {
+	clusterConfig := &cluster.Config{
```

Comment:
Since most of the variables map directly to the configuration, use a single configuration variable and populate it directly from command line:
```go
var clusterConfig cluster.Config
// ...
func init() {
	stolonctlCmd.PersistentFlags().StringVar(&clusterConfig.KubeConfig, "kubeconfig", "", "Kubernetes client config file")
	// and so on...
}
```
This will also make it easier to provide defaults later as suggested by Kevin.

Would also make sense to come up with a `CheckAndSetDefaults` method on the configuration to ensure parameter validity.